docs/usage.md

```diff
@@ -516,12 +548,29 @@ you can estimate crash severity for program built without sanitizers.
 You can set environment variable `RUST_BACKTRACE=(1|full)` for `casr-afl`. This
 variable may be used by [casr-san](#casr-san).
 
+Sharpfuzz example:
+
+    $ cp -r casr/tests/casr_tests/csharp/test_casr_afl_csharp /tmp/test_casr_afl_csharp
+    $ cp -r casr/tests/casr_tests/csharp/test_casr_afl_csharp_module /tmp/test_casr_afl_csharp_module
+    $ dotnet publish -o /tmp/test_casr_afl_csharp/bin
+    $ casr-afl -i casr/tests/casr_tests/casrep/afl-out-sharpfuzz -o casr/tests/tmp_tests_casr/casr_afl_csharp_out
+
+Sharpfuzz example (with --ignore-cmdline):
+
+    $ cp -r casr/tests/casr_tests/csharp/test_casr_afl_csharp /tmp/test_casr_afl_csharp
+    $ cp -r casr/tests/casr_tests/csharp/test_casr_afl_csharp_module /tmp/test_casr_afl_csharp_module
+    $ dotnet build /tmp/test_casr_afl_csharp/test_casr_afl_csharp.csproj
+    $ casr-afl --ignore-cmdline -i casr/tests/casr_tests/casrep/afl-out-sharpfuzz -o casr/tests/tmp_tests_casr/casr_afl_csharp_out -- dotnet run --no-build --project /tmp/test_casr_afl_csharp/test_casr_afl_csharp.csproj @@
+
+**NOTE:** if you run `casr-afl` for Sharpfuzz pipeline using `--ignore-cmdline` with `dotnet run`, build
+your project before (via `dotnet build` or `dotnet publish`) and specify `--no-build` option for `dotnet run`.
+
 ## casr-libfuzzer
 
 Triage crashes found by libFuzzer based fuzzer
 (C/C++/go-fuzz/Atheris/Jazzer/Jazzer.js/jsfuzz)
 
-    Usage: casr-libfuzzer [OPTIONS] --output <OUTPUT_DIR> [-- <ARGS>...]
+    Usage: casr-libfuzzer [OPTIONS] --output <OUTPUT_DIR> -- <ARGS>...
 
     Arguments:
       [ARGS]...  Add "-- ./fuzz_target <arguments>"
```

Comment:
I added `.required(true)` parameter to all casr binaries that needs ARGS to be without braces.
`.last(true)` parameter has a note: `NOTE: This will change the usage string to look like $ prog [OPTIONS] [-- <ARG>] if ARG is marked as .last(true).`
That is why we need to remove these braces from here and above.